pkgs/by-name/li/libkrun-efi/package.nix

```diff
@@ -0,0 +1,114 @@
+{
+  cargo,
+  fetchFromGitHub,
+  fetchurl,
+  fixDarwinDylibNames,
+  lib,
+  libepoxy,
+  libkrun-efi,
+  moltenvk,
+  pkg-config,
+  rustc,
+  rustPlatform,
+  rutabaga_gfx,
+  nix-update-script,
+  stdenv,
+  buildPackages,
+  meson,
+  ninja,
+  vulkan-headers,
+  withGpu ? true,
+}:
+let
+  virglrenderer = stdenv.mkDerivation (finalAttrs: {
```

Comment:
Can we use overrideAttrs here instead? Would that make sense?

Reply:
My understanding (maybe incomplete) is that using it in-tree introduces hard to untangle inter-dependencies without original derivation maintainers' consent.